R/process_data_sets_kara.R

```diff
@@ -0,0 +1,26 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+#reading in csv
+nba_elo <- read_csv("https://projects.fivethirtyeight.com/nba-model/nba_elo.csv")
+nba_elo_latest <- read_csv("https://projects.fivethirtyeight.com/nba-model/nba_elo_latest.csv")
+
+#tidying and taming data for nba_elo
+nba_elo %>% clean_names()
+gsub(x = colnames(nba_elo), pattern = "-", replacement = "_")
+nba_elo[13:22] <- as.numeric(as.character(nba_elo[13:22]))
```

Comment:
Not sure how @rudeboybert will feel about this use of hard-coded indices. I guess it's OK since these files are static. It would be more expressive to use `mutate_at()` or something like that.

Reply:
thank you! I changed it to a mutate_if statement.

Reply:
Yep, never hardcode anything. thanks